extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts

```diff
@@ -0,0 +1,371 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import * as azdataExt from 'azdata-ext';
+import * as loc from '../../../localizedConstants';
+import { IconPathHelper, cssStyles } from '../../../constants';
+import { DashboardPage } from '../../components/dashboardPage';
+import { convertToGibibyteString } from '../../../common/utils';
+import { MiaaModel } from '../../../models/miaaModel';
+
+export class MiaaComputeAndStoragePage extends DashboardPage {
+
+	private configurationContainer?: azdata.DivContainer;
+	private coresLimitBox?: azdata.InputBoxComponent;
+	private coresRequestBox?: azdata.InputBoxComponent;
+	private memoryLimitBox?: azdata.InputBoxComponent;
+	private memoryRequestBox?: azdata.InputBoxComponent;
+
+	private discardButton?: azdata.ButtonComponent;
+	private saveButton?: azdata.ButtonComponent;
+
+	private saveArgs: {
+		coresLimit?: string,
+		coresRequest?: string,
+		memoryLimit?: string,
+		memoryRequest?: string
+	} = {};
+
+	private readonly _azdataApi: azdataExt.IExtension;
+
+	constructor(protected modelView: azdata.ModelView, private _miaaModel: MiaaModel) {
+		super(modelView);
+		this._azdataApi = vscode.extensions.getExtension(azdataExt.extension.name)?.exports;
+
+		this.initializeConfigurationBoxes();
+
+		this.disposables.push(this._miaaModel.onConfigUpdated(
+			() => this.eventuallyRunOnInitialized(() => this.handleServiceUpdated())));
+	}
+
+	protected get title(): string {
+		return loc.computeAndStorage;
+	}
+
+	protected get id(): string {
+		return 'miaa-compute-and-storage';
+	}
+
+	protected get icon(): { dark: string; light: string; } {
+		return IconPathHelper.computeStorage;
+	}
+
+	protected get container(): azdata.Component {
+		const root = this.modelView.modelBuilder.divContainer().component();
+		const content = this.modelView.modelBuilder.divContainer().component();
+		root.addItem(content, { CSSStyles: { 'margin': '20px' } });
```

Comment:
Shouldn't this be defined in cssStyles constant defined in imported 'constants'?

Reply:
Not worth it for a one-off definition